apps/prairielearn/src/middlewares/content-security-policy.ts

```diff
@@ -12,6 +12,17 @@ router.all('/*', function (req, res, next) {
   // Added for backwards compatibility with older browsers.
   res.header('X-Frame-Options', 'DENY');
 
+  const originalResType: typeof res.type = res.type.bind(res);
+  res.type = (type) => {
```

Comment:
This won't work for calls like `res.setHeader('Content-Type', ...)`, of which we have at least one:

https://github.com/PrairieLearn/PrairieLearn/blob/e1b0807810588231850b4ca1f57dcb2206fa6de4/apps/prairielearn/src/pages/submissionFile/submissionFile.ts#L77

While we could update this to monkey-patch `setHeader` or update the above callsite to use `res.type` instead, it's probably more robust to use something like https://github.com/jshttp/on-headers to handle this in a fully generic way.

Reply:
Done in cb9680e29.